catkin_ws/src/auv_msgs/msg/PingerBearing.msg

```diff
@@ -1,7 +1,5 @@
-geometry_msgs/Vector3 pinger1_bearing
-geometry_msgs/Vector3 pinger2_bearing
-geometry_msgs/Vector3 pinger3_bearing
-geometry_msgs/Vector3 pinger4_bearing
+int32 frequency_index
```

Comment:
I think it would be clearer to keep it as an overall frequency and not an index, will prevent mistakes in the future if we decide to change the order or whatever and also itll make sure the message is more or less standalone

Reply:
yeah but then we have to keep loading the same parameters multiple times, turning them into a list and then getting the index. This way, you can skip steps

Reply:
I'm confused at what level this helps us gain steps? When we receive the message, we'll need to load an array and index it to know which frequency the index represents. If we use indexes we'll need to maintain an array of frequencies across the repo that will have to change based on the order of tasks, if they change frequencies, etc. If we keep everything in terms of frequencies, we don't have to keep track of any ordering, we can just know which direction certain frequencies point, and then just have a ros param for each task that indicates its frequency.